src/pool/srv_layout.c

```diff
@@ -96,4 +117,10 @@ ds_pool_prop_default_fini(void)
 		D_DEBUG(DB_MGMT, "Freeing default ACL pool prop\n");
 		D_FREE(entry->dpe_val_ptr);
 	}
+
+	entry = daos_prop_entry_get(&pool_prop_default, DAOS_PROP_PO_POLICY);
+	if (entry != NULL) {
+		D_DEBUG(DB_MGMT, "Freeing default policy pool prop\n");
+		D_FREE(entry->dpe_val_ptr);
+	}
```

Comment:
D_FREE will log anyway.
```suggestion
	if (entry != NULL)
		D_FREE(entry->dpe_val_ptr);
```